includes/controls/dimensions.php

```diff
@@ -84,55 +97,48 @@ protected function get_default_settings() {
 	 * @access public
 	 */
 	public function content_template() {
-		$dimensions = [
-			'top' => esc_html__( 'Top', 'elementor' ),
-			'right' => esc_html__( 'Right', 'elementor' ),
-			'bottom' => esc_html__( 'Bottom', 'elementor' ),
-			'left' => esc_html__( 'Left', 'elementor' ),
-		];
+		$class_name = $this->get_singular_name();
 		?>
 		<div class="elementor-control-field">
 			<label class="elementor-control-title">{{{ data.label }}}</label>
 			<?php $this->print_units_template(); ?>
 			<div class="elementor-control-input-wrapper">
-				<ul class="elementor-control-dimensions">
+				<ul class="elementor-control-<?php echo esc_attr( $class_name ); ?>s">
 					<?php
-					foreach ( $dimensions as $dimension_key => $dimension_title ) :
+					foreach ( $this->dimensions() as $dimension_key => $dimension_title ) :
 						?>
-						<li class="elementor-control-dimension">
+						<li class="elementor-control-<?php echo esc_attr( $class_name ); ?>">
 							<input id="<?php $this->print_control_uid( $dimension_key ); ?>" type="text" data-setting="<?php
 								// PHPCS - the variable $dimension_key is a plain text.
 								echo $dimension_key; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 							?>" placeholder="<#
-								placeholder = view.getControlPlaceholder();
-								if ( _.isObject( placeholder ) ) {
-									if ( ! _.isUndefined( placeholder.<?php
+									placeholder = view.getControlPlaceholder();
+									if ( _.isObject( placeholder ) && ! _.isUndefined( placeholder.<?php
 										// PHPCS - the variable $dimension_key is a plain text.
 										echo $dimension_key; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 									?> ) ) {
-										print( placeholder.<?php
+											print( placeholder.<?php
```

Comment:
Indentation.

Reply:
@hein-obox This was a bit of a refactor, so it just looks like the code moved unnecessarily in git.